### PR TITLE
DOCS Add new user help for file permissions and secure files/folders

### PR DESCRIPTION
**Pull request by sachajudd**
Fixes: https://github.com/silverstripe/silverstripe-userforms/issues/971
Relies on:  https://github.com/silverstripe/silverstripe-userforms/pull/980 to be merged.

Includes updates to readme for new naming conventions and updated author list.

### CODE REVIEW

**Comment by sachajudd** [comment]
Good catch @bergice 😆

File: docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md
Comment (bergice) [inline docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md]: Relative links are preferred so we don't have to update them later on:

```suggestion
[Form submissions, File Upload Field](/optional_features/forms/form-submissions/#file-upload-field).
```

File: docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md
Comment (bergice) [inline docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md]: Relative links are preferred so we don't have to update them later on:

```suggestion
* Below [relevant folder](/optional_features/forms/form-submissions/#changing-folders-for-individual-fields) select dropdowns
```

**Review by bergice — CHANGES_REQUESTED** [review]

File: docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md
Comment (sachajudd) [reply docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md]: Yeah cool thanks, couldn't remember how we did them so thought this would come up 👍

**Comment by sachajudd** [comment]
Updated @bergice

File: docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md
Comment (brynwhyman) [inline docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md]:
```suggestion
To help reduce the risk of privacy breaches of sensitive files, particularly user submitted data, Silverstripe CMS has some helpful defaults to
```

File: docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md
Comment (brynwhyman) [inline docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md]:
```suggestion
on your site, your form submissions are more likely to contain [sensitive information](/optional_features/forms/#data-protection-and-privacy) which may require heightened
levels of compliance to ensure there are not any accidental data breaches.
```
I think PII is only one example of 'sensitive information'.

File: docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md
Comment (brynwhyman) [inline docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md:56]: I've made a [suggestion in the related PR](https://github.com/silverstripe/silverstripe-userforms/pull/980#issuecomment-654449906) that we move this to a separate page, if that's accepted we'll need to update this link

File: docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md
Comment (brynwhyman) [inline docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/09_File_Permissions.md]:
```suggestion
* File/folder details and folder heading titles
```

**Review by brynwhyman — APPROVED** [review]
I won't merge this (will leave that to @bergice) but LGTM

**Pull request merged**
